addons/survey/survey.py

```diff
@@ -486,8 +486,14 @@ def action_test_survey(self, cr, uid, ids, context=None):
             'url': self.read(cr, uid, ids, ['public_url'], context=context)[0]['public_url'] + "/phantom"
         }
 
-
-
+    def action_survey_user_input(self, cr, uid, ids, context=None):
+        action_rec = self.pool['ir.model.data'].xmlid_to_object(cr, uid, 'survey.action_survey_user_input', context=context)
+        action = action_rec.read()[0]
```

Comment:
This seems useless and inefficient.

Why not returning something like 
         return {
             'type': 'ir.actions.act_window',
             'blabla'
         }